helm/cassandra/values.yaml

```diff
@@ -5,12 +5,13 @@
 replicaCount: 3
 
 image:
-  cassandraRepository: gcr.io/cassandra-operator/cassandra-dev
+  cassandraRepository: gcr.io/cassandra-operator/cassandra
   sidecarRepository: gcr.io/cassandra-operator/cassandra-sidecar-dev
```

Comment:
@benbromhead not sure if we should be removing `-dev` from the sidecar image repo name too?

Reply:
we probably should.

Reply:
sorry, i couldn't find the dev image, so i switched this for testing. I eventually made my own images. i'll revert.